src/user-management/user-management.ts

```diff
@@ -315,6 +318,25 @@ export class UserManagement {
     });
   }
 
+  async authenticateWithCodeAndVerifier(
+    payload: AuthenticateWithCodeAndVerifierOptions,
+  ): Promise<AuthenticationResponse> {
+    const { session, ...remainingPayload } = payload;
+
+    const { data } = await this.workos.post<
+      AuthenticationResponseResponse,
+      SerializedAuthenticateWithCodeAndVerifierOptions
```

Comment:
not relevant to this PR: but i really wish we would move away from manually building out all these camelCase->snake_case serializers/deserializers.